dash/include/dash/internal/TypeInfo.h

```diff
@@ -16,6 +16,13 @@ std::string typestr(const T & obj) {
          );
 }
 
+template <class T>
+std::string typestr() {
+  return dash::internal::demangle(
+           typeid(T).name()
+         );
+}
+
```

Comment:
Documentation! Otherwise we will re-invent this in a couple of month, as no one knows that this exists.